### ISSUE TITLE
ContrastSplatterDemo is broken

### ISSUE THREAD

**Issue by DavidBrainard** [spitschan/SilentSubstitutionToolbox#6]
This might be because it refers to 'Hemo' cone types, which no longer exist.

**Comment by spitschan** [comment]
`ContrastSplatterDemo` is a deprecated function since it does not use the Asano model for parametric/stochastic variation of the cone fundamental parameters. I may fit here for legacy purposes, and we may elect to remove it from the code base at some point.

**Comment by spitschan** [comment]
This should now work again. The state-of-the-art remains the Asano model and the stochastic resampling and what we've implemented for the bioRxiv paper. I've fixed up this code so that it runs. The only thing that no longer work is calculating the 95% and 99% confidence ellipses because of some re-definitions of the functions that calculate the error ellipses. I've commented out that code. At some point, I expect this function to go away, with the @SSTReceptorObject being the supported way to calculate splatter.
